bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/obscomp/SPInstObsComp.java

```diff
@@ -410,4 +412,31 @@ public void restoreScienceDetails(final SPInstObsComp oldData) {
     public Instrument getInstrument() {
         return Instrument.fromComponentType(getType()).getValue();
     }
+
+    // REL-1678: 7 seconds DHS write overhead
+    // REL-1934: 10 seconds DHS write overhead for F2
+
+    public static final Map<Instrument, PlannedTime.CategorizedTime> DHS_WRITE_TIMES;
+
+    static {
+        Map<Instrument, Duration> m =
+            Arrays.stream(Instrument.values()).collect(Collectors.toMap(Function.identity(), i -> Duration.ofSeconds(7)));
+
+        m.put(Instrument.Flamingos2, Duration.ofSeconds(10));
+        m.put(Instrument.GmosNorth,  Duration.ofSeconds(10));
+        m.put(Instrument.GmosSouth,  Duration.ofSeconds(10));
+        m.put(Instrument.Gnirs,      Duration.ofMillis(8500));
+        m.put(Instrument.Nifs,       Duration.ofMillis(9800));
+        m.put(Instrument.Niri,       Duration.ofMillis(5070));
+
+        DHS_WRITE_TIMES = Collections.unmodifiableMap(
+          m.entrySet()
+           .stream()
+           .collect(Collectors.toMap(Map.Entry::getKey, e -> PlannedTime.CategorizedTime.apply(PlannedTime.Category.DHS_WRITE, e.getValue().toMillis())))
+        );
+    }
+
+    public final PlannedTime.CategorizedTime DHS_WRITE_TIME =
+        DHS_WRITE_TIMES.get(getInstrument());
+
```

Comment:
This block is the key change.  It pulls together the DHS write times for all of the instruments and provides a `DHS_WRITE_TIME` value that can be used from any `SPInstObsComp` to obtain the instrument-specific write time prediction.